pshtt/pshtt.py

```diff
@@ -1015,28 +1022,38 @@ def did_domain_error(domain):
     )
 
 
-def fetch_preload_pending():
+def load_preload_pending():
```

Comment:
Standardized on `load_` as the prefix for these 3 loading/caching functions, since they were split among `load_`, `create_`, and `fetch_` before, which seemed slightly confusing.